mcclim.asd

```diff
@@ -66,10 +66,11 @@ interface management system."
 (defsystem #:mcclim/looks
   :depends-on (#:clim
                #-(or mcclim-beagle mcclim-ugly)
-               #:mcclim-clx/pretty              #| adds truetype        |#
-               #+mcclim-ugly #:mcclim-clx       #| raw clim-clx backend |#
-               #-mcclim-beagle #:mcclim-clx-fb  #| experimental backend |#
-               #+mcclim-beagle #:mcclim-beagle  #| OSX native (clozure) |#
+               #+old-fonts #:mcclim-clx/truetype  #| adds truetype        |#
+               #:mcclim-clx/freetype              #| adds freetype        |#
```

Comment:
if I understand correctly, freetype requires groveller (hence C compiler) at runtime (eval-when) – this can't be default in such case.

Reply:
It does require the groveller. It's theoretically possible to build this without the groveller. But I'd rather concentrate on that once everything else works well.

Reply:
as I said, this can't be default, please adjust your PR to take into account that.